src/editor.rs

```diff
@@ -350,17 +352,17 @@ pub struct Editor<'arena, Node: Ast<'arena>> {
     /// The current contents of the command buffer
     command: String,
     /// The configured key map
-    keymap: KeyMap,
+    keymap: normal_mode::KeyMap,
```

Comment:
I think it'd probably make sense to `use normal_mode::{KeyMap, Action, parse_command}` at the top of this file - it'll remove the changes to the main module in the file, as well as clearing the code up a bit.  For the unit tests, it's probably fine.